src/main/java/frc/robot/subsystems/IntakeSystem.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSystem extends SubsystemBase {
+    private CANSparkMax motor;
+    private DigitalInput limitSwitch;
+
+    public IntakeSystem(){
+        motor = new CANSparkMax(RobotMap.INTAKE_MOTOR, CANSparkLowLevel.MotorType.kBrushless);
+        limitSwitch = new DigitalInput(RobotMap.INTAKE_LIMIT_SWITCH);
+    }
+
+    public void out(){
+        motor.set(0.6);
```

Comment:
Declare `0.6` as a constant in the class (define it above the class members). This will make it clearer to other users about the meaning of the value, and will allow us to use it multiple times.

something like
```
private static final double DEFAULT_ROTATE_SPEED = 0.6
```

Use this in `in` method as well.

Reply:
done